packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8002,25 +8027,33 @@ describe('plugin-meetings', () => {
   });
 
   describe('#getPermissionTokenTimeLeftInSec', () => {
-    it('should return undefined if permissionTokenTtl is undefined', () => { 
-      assert.equal(meeting.getPermissionTokenTimeLeftInSec(), undefined)
+    let now;
+    let clock;
+
+    beforeEach(() => {
+      now = Date.now();
+
+      // mock `new Date()` with constant `now`
+      clock = sinon.useFakeTimers(now);
     });
 
-    it('should return the expected positive permissionTokenTimeLeft', () => { 
-      const now = Date.now();
-      sinon.stub(Date, 'now').returns(now);
+    afterEach(() => {
+      clock.restore();  
+    })
+
+    it('should return undefined if exp is undefined', () => { 
+      assert.equal(meeting.getPermissionTokenTimeLeftInSec(), undefined)
+    });
 
+    it('should return the expected positive exp', () => { 
       // set permission token as now + 1 sec
-      meeting.permissionTokenTtl = now + 1000;
+      meeting.permissionTokenPayload = {exp: (now + 1000).toString()};
```

Comment:
nitpick: This doesn't need to be made a string as it is actually a number in the permissionToken, doesn't matter too much though as this tests the conversion!